### ISSUE TITLE
wmic and Windows 10 Version 2004

### ISSUE THREAD

**Issue by erSitzt** [inverse-inc/packetfence#6014]
**Describe the bug**
Not sure if this is already a problem, but i stumbeled upon an issue with the linux version of wmic when we added newer Windows-Systems to out nagios monitoring, so i was curious if you already know of this and maybe have a solution

Trying to query newer Windows Installs wmic fails with this error
```
[wmi/wmic.c:196:main()] ERROR: Login to remote object.
NTSTATUS: NT code 0x80010111 - NT code 0x80010111
```
And i found this Post by the creators of the wmi check plugin for nagios
https://edcint.co.nz/checkwmiplus/expression-of-interest-redevelopment-of-wmic-client/
https://edcint.co.nz/checkwmiplus/help-needed-wmic-broken-as-of-windows-10-version-2004/

**To Reproduce**
Try this with any newer Win 10 (and/or Server) install
`wmic -U [domain/]adminuser%password //host "select * from Win32_ComputerSystem"`

If this is irrelevant for packetfence feel free to close this issue, i just want to spread the word because i see this as a problem with some impact for everything regarding linux to windows management scripting....

**Comment by julsemaan** [comment]
I'd say lets leave this opened if we hit the issue then that will give us a good place to start our investigation

**Comment by nqb** [comment]
Interesting thread on this topic: https://docs.microsoft.com/en-us/answers/questions/146551/wmic-stopped-working-on-windows-10-2004.html

**Comment by nqb** [comment]
I got same behavior on a Windows 10 2004. I tried to look at Event viewer and compare with a valid WMI session on a Windows 2016 server using Wireshak. 

It looks like something change in DCE/RPC response sent by Windows 10 device. 

The DCOM operation RemoteActivation returned by Windows 10 device has a different format which could explain why `wmic` is not able to understand it.

Anyway, we already discuss in the past to drop `wmic` in place of `impacket`. Perhaps, we will need to do that.
